iOS/src/toga_iOS/widgets/base.py

```diff
@@ -13,6 +13,15 @@ def __init__(self, interface):
         self._container = None
         self.constraints = None
         self.native = None
+
+        # Set default background color
+        try:
+            # systemBackgroundColor() was introduced in iOS 13
```

Comment:
We no longer need this branch; as a result of the PEP 730 changes, iOS 13 is the minimum supported iOS version. The [iOS platform docs](https://toga.readthedocs.io/en/latest/reference/platforms/iOS.html#prerequisites) should also be updated to reflect this.

Reply:
Thanks, I've removed that branch.